src/pages/iou/request/step/IOURequestStepWaypoint.tsx

```diff
@@ -232,31 +232,29 @@ function IOURequestStepWaypoint({
 
 IOURequestStepWaypoint.displayName = 'IOURequestStepWaypoint';
 
-// eslint-disable-next-line rulesdir/no-negated-variables
-const IOURequestStepWaypointWithWritableReportOrNotFound = withWritableReportOrNotFound(IOURequestStepWaypoint);
-// eslint-disable-next-line rulesdir/no-negated-variables
-const IOURequestStepWaypointWithFullTransactionOrNotFound = withFullTransactionOrNotFound(IOURequestStepWaypointWithWritableReportOrNotFound);
+export default withFullTransactionOrNotFound(
```

Comment:
You changed the order of HOC here, please make sure it doesn't introduce any regressions

Reply:
I have't found any issue while testing, but just in case I've swapped it.
Thanks!